src/utils.rs

```diff
@@ -0,0 +1,22 @@
+use ansi_term::{Color, Style};
+
+/// Print a warning to stderr. If `enable_styles` is true, then a yellow
+/// message will be printed.
+pub fn print_warning(enable_styles: bool, message: &str) {
+    print_msg(enable_styles, message, "Warning: ", Color::Yellow);
+}
+
+/// Print an error to stderr. If `enable_styles` is true, then a red message
+/// will be printed.
+pub fn print_error(enable_styles: bool, message: &str) {
+    print_msg(enable_styles, message, "Error: ", Color::Red);
+}
```

Comment:
I don't think that a function named `print_*` should exit the process, there could be one like called `die` or so that uses the `print_critical` function

Reply:
True. Let's skip that for now.